care/emr/signals/charge_item_def/activity_definition.py

```
@@ -0,0 +1,49 @@
+            instance,
+            _OLD_CHARGE_ITEM_DEFS_IDS,
+            list(old_instance.charge_item_definitions or []),
+        )
+        setattr(instance, _OLD_DELETED, old_instance.deleted)
+    else:
+        setattr(instance, _OLD_CHARGE_ITEM_DEFS_IDS, [])
+        setattr(instance, _OLD_DELETED, False)
+
+
+@receiver(post_save, sender=ActivityDefinition)
+def update_separately_billable_activity(sender, instance, created, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `sender`

(ARG001)